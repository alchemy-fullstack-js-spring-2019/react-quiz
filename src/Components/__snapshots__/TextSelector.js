import React, { PureComponent } from 'react';
import PropTypes from 'prop-types';

export default class TextSelector extends 