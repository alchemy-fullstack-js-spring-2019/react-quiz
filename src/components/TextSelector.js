import React from 'react';
import PropTypes from 'prop-types';

export default function TextSelector({ text, selectText }) {
  return (
    <input name="text" value={text} onChange={selectText} />
  );
}
