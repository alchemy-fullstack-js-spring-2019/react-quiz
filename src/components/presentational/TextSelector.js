import React from 'react';
import PropTypes from 'prop-types';

export default function TextSelector({ text, handleChange }) {
  return <input name="text" value={text} onChange={handleChange} />;
}

TextSelector.propTypes = {
  text: PropTypes.string,
  handleChange: PropTypes.func
};
