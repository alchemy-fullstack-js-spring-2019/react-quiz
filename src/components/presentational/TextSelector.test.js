import React from 'react';
import { shallow } from 'enzyme';
import TextSelector from './TextSelector';

describe('TextSelector Test', () => {
  it('renders TextSelector', () => {
    const wrapper = shallow(<TextSelector />);
    expect(wrapper).toMatchSnapshot();
  });
});
