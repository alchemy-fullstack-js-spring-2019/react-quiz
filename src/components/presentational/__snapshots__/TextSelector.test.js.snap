// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextSelector Test renders TextSelector 1`] = `
<input
  name="text"
/>
`;
